packages/common-all/src/constants/configs/compat.ts

```diff
@@ -3,5 +3,5 @@ export const CONFIG_TO_MINIMUM_COMPAT_MAPPING: { [key: number]: string } = {
   2: "0.63.0", // config consolidation (commands namespace)
   3: "0.65.0", // config consolidation (workspace namespace)
   4: "0.70.0", // config consolidation (preview namespace)
-  5: "0.81.0", // config consolidation (publishing namespace) TODO: FIX THIS BEFORE GENERAL RELEASE
+  // 5: "0.83.0", // config consolidation (publishing namespace) TODO: FIX THIS BEFORE GENERAL RELEASE
```

Comment:
I've commented this out because adding this compat mapping would prevent users from keeping the v4 config and still use the cli for publishing.

If we want to force the migration I can re-enable it.

Reply:
looks good, can you also add this comment in the code?